osquery/registry/registry_factory.cpp

```diff
@@ -89,7 +89,9 @@ Status RegistryFactory::addBroadcast(const RouteUUID& uuid,
           VLOG(1) << "Extension " << uuid
                   << " has duplicate plugin name: " << item.first
                   << " in registry: " << registry.first;
-          return Status(1, "Duplicate registry item: " + item.first);
+          return Status(1,
+                        "Registry item " + item.first +
+                            " conflicts with another item of the same name.");
```

Comment:
Why this change?

Reply:
Made the message a bit clearer. We had a customer hitting issues with duplicate table names, so we decided to look into this area and see if we could do something.

Reply:
Okay? I think "Duplicate" is pretty clear, and easy to grep for in logs. But I don't think I've hit it, and don't actually have an issue with the rewording.